docs/guides/user.rst

```diff
@@ -0,0 +1,12 @@
+Manticore
+----------
+
+Mainticore implements few application in the application layer. Once `kernel.elf`
```

Comment:
s/Mainticore/Manticore/

Reply:
Something like: "The Manticore source tree has some example applications showing how to run applications on top of the OS."